Description: Rename checkXML to resolve conflict. 
 checkXML conflicts with kde4 checkXML so I renamed checkXML to checkXML5.
 .
 kdoctools (4.95.0-0ubuntu1~ubuntu14.04~ppa4) trusty; urgency=low
 .
   * Revision 2
Author: Scarlett Clark <scarlett@scarlettgatelyclark.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- kdoctools-4.95.0.orig/CMakeLists.txt
+++ kdoctools-4.95.0/CMakeLists.txt
@@ -86,11 +86,11 @@ install(FILES
 )
 
 
-configure_file(checkXML.in.cmake ${CMAKE_CURRENT_BINARY_DIR}/checkXML )
+configure_file(checkXML5.in.cmake ${CMAKE_CURRENT_BINARY_DIR}/checkXML5 )
 include(ConfigureChecks.cmake)
 configure_file(config-kdoctools.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config-kdoctools.h )
 
-install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/checkXML DESTINATION ${BIN_INSTALL_DIR} )
+install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/checkXML5 DESTINATION ${BIN_INSTALL_DIR} )
 
 install( FILES
     KF5DocToolsMacros.cmake
--- /dev/null
+++ kdoctools-4.95.0/checkXML5.in.cmake
@@ -0,0 +1,6 @@
+#! /bin/sh
+
+prefix=${CMAKE_INSTALL_PREFIX}
+exec_prefix=${CMAKE_INSTALL_PREFIX}/bin
+exit `${CMAKE_INSTALL_PREFIX}/bin/meinproc5 --check --stdout $@ > /dev/null`
+
